esp/services/ws_workunits/ws_workunitsQuerySets.cpp

```diff
@@ -420,6 +420,36 @@ void QueryFilesInUse::loadTargets(IPropertyTree *t, unsigned flags)
     }
 }
 
+void QueryFilesInUse::updateUsers()
+{
+    SCMStringBuffer target;
+
+#ifdef _CONTAINERIZED
```

Comment:
This code is now essentially the same for container/non container, and should be simplified.

Reply:
Will do